raft/quorum/majority.go

```diff
@@ -102,6 +102,16 @@ func (c MajorityConfig) Describe(l AckedIndexer) string {
 	return buf.String()
 }
 
+// Slice returns the MajorityConfig as a sorted slice.
+func (c MajorityConfig) Slice() []uint64 {
+	var sl []uint64
```

Comment:
Should we do the same thing as `String` method? `sl := make([]uint64, 0, len(c))`?

Reply:
I want to return `nil` when `len(c) == 0`. But we have similar code in so many places right now that I'll take a (later) pass where I factor out some helpers for `map[uint64]struct{}{}`.